### ISSUE TITLE
When I add "Model group" in "template", there is an error "java.lang.NullPointerException: No description available."

### ISSUE THREAD

**Issue by xwwunknowbody** [alkacon/opencms-core#553]
When I add "Model group" in "template", there is an error "java.lang.NullPointerException: No description available."

My version is 10.5.3

After testing, 10.5.2 does not have this problem
![Image of Yaktocat](https://c1.staticflickr.com/5/4574/38119587115_485209350d_b.jpg)
![Image of Yaktocat](https://c1.staticflickr.com/5/4647/38290626304_b5d2008242_b.jpg)

```
12 十二月 2017 21:09:47,360 ERROR [ org.opencms.gwt.CmsGwtService: 183] 
java.lang.NullPointerException
	at org.opencms.ade.containerpage.CmsModelGroupHelper.getModelReplacementElement(CmsModelGroupHelper.java:908)
	at org.opencms.ade.containerpage.CmsModelGroupHelper.prepareforModelGroupContent(CmsModelGroupHelper.java:342)
	at org.opencms.ade.containerpage.CmsContainerpageService.getElements(CmsContainerpageService.java:2134)
	at org.opencms.ade.containerpage.CmsContainerpageService.getElementsData(CmsContainerpageService.java:736)
	at sun.reflect.GeneratedMethodAccessor224.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:587)
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:333)
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:303)
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373)
	at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	at org.opencms.gwt.CmsGwtService.service(CmsGwtService.java:194)
	at org.opencms.main.OpenCmsCore.invokeGwtService(OpenCmsCore.java:1719)
	at org.opencms.main.OpenCmsServlet.doGet(OpenCmsServlet.java:158)
	at org.opencms.main.OpenCmsServlet.doPost(OpenCmsServlet.java:177)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:230)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:192)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
	at org.opencms.jsp.jsonpart.CmsJsonPartFilter.doFilter(CmsJsonPartFilter.java:281)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:192)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:726)
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:469)
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:394)
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:311)
	at org.opencms.main.OpenCmsUrlServletFilter.doFilter(OpenCmsUrlServletFilter.java:127)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:192)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:198)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:474)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:624)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:349)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:783)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:798)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1434)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:745)
12 十二月 2017 21:09:47,427 ERROR [ org.opencms.gwt.CmsLogService:  66] 客户端日志(Host 106.41.33.15, Address 106.41.33.15, Ticket 1513084195151): null
org.opencms.ade.containerpage.CmsModelGroupHelper.getModelReplacementElement(CmsModelGroupHelper.java:908)
org.opencms.ade.containerpage.CmsModelGroupHelper.prepareforModelGroupContent(CmsModelGroupHelper.java:342)
org.opencms.ade.containerpage.CmsContainerpageService.getElements(CmsContainerpageService.java:2134)
org.opencms.ade.containerpage.CmsContainerpageService.getElementsData(CmsContainerpageService.java:736)
sun.reflect.GeneratedMethodAccessor224.invoke(Unknown Source)
sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
java.lang.reflect.Method.invoke(Method.java:498)
com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:587)
com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:333)
com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:303)
com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373)
com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
org.opencms.gwt.CmsGwtService.service(CmsGwtService.java:194)
org.opencms.main.OpenCmsCore.invokeGwtService(OpenCmsCore.java:1719)
org.opencms.main.OpenCmsServlet.doGet(OpenCmsServlet.java:158)
org.opencms.main.OpenCmsServlet.doPost(OpenCmsServlet.java:177)
javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:230)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:192)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
org.opencms.jsp.jsonpart.CmsJsonPartFilter.doFilter(CmsJsonPartFilter.java:281)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:192)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:726)
org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:469)
org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:394)
org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:311)
org.opencms.main.OpenCmsUrlServletFilter.doFilter(OpenCmsUrlServletFilter.java:127)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:192)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:198)
org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:474)
org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140)
org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:624)
org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:349)
org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:783)
org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:798)
org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1434)
org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
java.lang.Thread.run(Thread.java:745)
```

**Comment by pimonty** [comment]
Hi, when i try to drag a model in the template, the same error appear. 
Version: 10.5.3, Chrome  63.0.3239.108 (Build oficial) (64 bits)

after the error, sometimes when i press the rocket menu show the same error message. 

In adition, i try it in Firefox 57.0.1 but the same issue.

java.lang.NullPointerException: No description available.

Ticket Id: 1514487271883

java.lang.NullPointerException: null

org.opencms.ade.containerpage.CmsModelGroupHelper.getModelReplacementElement(CmsModelGroupHelper.java:908)
org.opencms.ade.containerpage.CmsModelGroupHelper.prepareforModelGroupContent(CmsModelGroupHelper.java:342)
org.opencms.ade.containerpage.CmsContainerpageService.getElements(CmsContainerpageService.java:2134)
org.opencms.ade.containerpage.CmsContainerpageService.getElementsData(CmsContainerpageService.java:736)
jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(NativeMethodAccessorImpl.java)
jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
java.lang.reflect.Method.invoke(Method.java:564)
com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:587)
com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:333)
com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:303)
com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373)
com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
org.opencms.gwt.CmsGwtService.service(CmsGwtService.java:194)
org.opencms.main.OpenCmsCore.invokeGwtService(OpenCmsCore.java:1719)
org.opencms.main.OpenCmsServlet.doGet(OpenCmsServlet.java:158)
org.opencms.main.OpenCmsServlet.doPost(OpenCmsServlet.java:177)
javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:292)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
org.opencms.jsp.jsonpart.CmsJsonPartFilter.doFilter(CmsJsonPartFilter.java:281)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:716)
org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:466)
org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:391)
org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:318)
org.opencms.main.OpenCmsUrlServletFilter.doFilter(OpenCmsUrlServletFilter.java:127)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:212)
org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:94)
org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:504)
org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:141)
org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:620)
org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:502)
org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1132)
org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:684)
org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1539)
org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1495)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
java.lang.Thread.run(Thread.java:844)

**Comment by pimonty** [comment]
The error begun when copy a model en opencms 10.5.3

path = /sites/default/apollo-demo/.content/modelgroup

this files are:  modelgroup-0001.html, modelgroup-0002.html, modelgroup-0003.html,  modelgroup-0004.html,  modelgroup-0005.html,  modelgroup-0006.html.  

If you want to restart the original files of the demo sample deleting your files copied, that not happen the error persists from any template when you try to drag some of this "modelgroup-000x.html".

**Comment by pimonty** [comment]
Confirm. Opencms 10.5.2 + JDK 8 + mysql 5.7  = No problems

**Comment by jgregorio0** [comment]
Good Morning! I experienced the same error.

It looks like CmsModelGroupHelper.getModelReplacementElement method throws exception because:
1- element.getFormatterId() returns null
2- when evaluating element.getFormatterId().equals(baseElement.getFormatterId())

I achieve a solution by adding the condition `element.getFormatterId() != null` (line 908):

`if (!baseElement.isCopyModel() &&
                element.getFormatterId() != null &&
                !element.getFormatterId().equals(baseElement.getFormatterId()))`

After this change I've built, replaced opencms.jar library and run under:
MySQL 5.7
JDK 1.7.0_80
OpenCms 10.5.3

Enjoy!

**Comment by BlueOtaku** [comment]
oh!

**Comment by jgregorio0** [comment]
After this change when drag & drop model group (copy type) from ADE and editing settings it throws error. It is required to reload page F5 after drop model group, then you can edit settings like normally.

**Comment by bzebedin** [comment]
I have the same problem. On a 10.5.2 installation it works without a problem.
On 10.5.3 (upgraded from 10.5.2) the same problem
On a complete fresh installation 10.5.3 the same problem.

**Comment by koronchik** [comment]
Hi, this problem only on 10.5.3.
To correct fix this problem I change line 908:
`if (!baseElement.isCopyModel() && element.getFormatterId().equals(baseElement.getFormatterId()))`
to 
`if (!baseElement.isCopyModel() && !baseElement.getFormatterId().equals(element.getFormatterId()))`

Adding the condition element.getFormatterId() != null (line 908) not enough.

**Comment by jgregorio0** [comment]
After this modification is throwing an error when we add any reused element and modify settings. Does it happen to you?

1- From ADE, Drag and Drop any reused element (layoutrow, 
2- After drop, edit settings, change any option and OK.
3- It throws attached error reading element settings from cache.
4- Reload page (F5)
5- Modify settings and it works fine.

```Error reading element settings from cache. Please reload the page.

Ticket Id: 1519980116610

org.opencms.main.CmsException: Error reading element settings from cache. Please reload the page.

org.opencms.ade.containerpage.CmsContainerpageService.getCachedElement(CmsContainerpageService.java:1852)
org.opencms.ade.containerpage.CmsContainerpageService.saveElementSettings(CmsContainerpageService.java:1391)
sun.reflect.NativeMethodAccessorImpl.invoke0(NativeMethodAccessorImpl.java)
sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
java.lang.reflect.Method.invoke(Method.java:606)
com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:587)
com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:333)
com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:303)
com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373)
com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
org.opencms.gwt.CmsGwtService.service(CmsGwtService.java:194)
org.opencms.main.OpenCmsCore.invokeGwtService(OpenCmsCore.java:1719)
org.opencms.main.OpenCmsServlet.doGet(OpenCmsServlet.java:158)
org.opencms.main.OpenCmsServlet.doPost(OpenCmsServlet.java:177)
javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:230)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:192)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
org.opencms.jsp.jsonpart.CmsJsonPartFilter.doFilter(CmsJsonPartFilter.java:281)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:192)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:726)
org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:469)
org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:394)
org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:311)
org.opencms.main.OpenCmsUrlServletFilter.doFilter(OpenCmsUrlServletFilter.java:127)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:192)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:198)
org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:108)
org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:472)
org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140)
org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:620)
org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:349)
org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:784)
org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:802)
org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1410)
org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
java.lang.Thread.run(Thread.java:745)
```

**Comment by koronchik** [comment]
No, I do not have this error, settings change successfully.

**Comment by tHerrmann** [comment]
This has been fixed in ranch_10_5_x.

**Comment by tHerrmann** [comment]
This has been fixed with release 10.5.4.
